documentation/source/user_section/tutorials/4/4-4.rst

```diff
@@ -0,0 +1,29 @@
+.. _mt-registraton-without-anat:
+
+4.4 Registering MT data directly to the template (Option 2)
+###########################################################
+
+In the case that you have only the MT data without the anatomical data, you can still perform registration. To do so, all you will need to do is apply the same vertebral labeling and template registration steps that were covered in :ref:`registration-to-template`.
+
+First, we create one or two labels in the metric space. For example, if you know that your FOV is centered at C3/C4 disc, then you can create a label automatically with:
```

Comment:
```suggestion
First, we create one or two labels in the metric space. For example, if you know that your field of view (FOV) is centered at C3/C4 disc, then you can create a label automatically with:
```

Reply:
Addressed here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3094/commits/b951cfacf8343be5742d1da68117c717ef6568e8 in `documentation/source/user_section/tutorials/registering-additional-contrasts/spinalcord-segmentation.rst`